tests/phpunit/README.md

```diff
@@ -0,0 +1,33 @@
+# PHPUnit Tests
+
+PHPUnit is the official testing framework chosen by the core team to test our PHP code.
+
+This documentation will assume you have a local development environment of your own choosing. For other testing workflows, such as Docker, see the [PHP: PHP Unit](https://make.wordpress.org/core/handbook/testing/automated-testing/phpunit/) handbook page.
+
+## Setup
+
+1. Check out the wordpress-develop repository
+
+       git clone git://develop.git.wordpress.org/ wordpress-develop
+       cd wordpress-develop
+
+2. Run `composer update --with-all-dependencies`
+3. Create a clean MySQL database and user. **DO NOT USE AN EXISTING DATABASE** or you will lose data, guaranteed.
+4. Copy `wp-tests-config-sample.php` to `wp-tests-config.php`, edit it to include your testing database credentials.
+5. Run the tests from the repository root:
+   - To execute a particular test:
+
+         $ vendor/bin/phpunit tests/phpunit/tests/test_case.php
+   - To execute all tests:
+
+         $ vendor/bin/phpunit
```

Comment:
```suggestion
- To execute a particular test:

vendor/bin/phpunit tests/phpunit/tests/test_case.php

- To execute all tests:

vendor/bin/phpunit

```
Suggest dropping the terminal prompt character. For these commands, please substitute the block formatting spaces by wrapping with three backticks for [consistency with other docs](https://github.com/WordPress/wordpress-develop/blob/552077a27589294b2cf6097205a4045585016281/README.md?plain=1#L50-L55) (I'm unsure how to add these within a backticked suggestion block 😅).